Tests/SpeziFoundationTests/RuntimeEnvironmentTests.swift

```diff
@@ -23,7 +22,7 @@ struct RuntimeEnvironmentTests {
         #endif
     }
     
-    @Test
+    @Test(.disabled(if: isLinux, "Skipped on Linux: runningInXCTest() calls NSClassFromString() which uses the Objective-C runtime."))
```

Comment:
Can we add an availability annotation to the implementation to avoid that it can even be called on other platforms than apple ones? We might even want to deprecate it @bauer-andreas?